lidarrmetadata/api.py

```diff
@@ -144,7 +128,50 @@ def get_artist_info(mbid):
     return jsonify(artist)
 
 
+@app.route('/album/<mbid>', methods=['GET'])
```

Comment:
Removing trailing slashes causes issues in Lidarr, when removing from the Lidarr side search fails. I don't think these were causing the issues with args.

Reply:
I think there's a chance we may still want to remove the trailing slash. If args are passed, having the trailing slash redirects `/album/<mbid>?key=value` to `/album/<mbid>/?key=value`. That may be alright, but it's a bit different than what I'm used to seeing.